File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoLayoutGrid.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Symbol;
import com.neome.api.meta.base.Types.EnumDefnLayoutGridKind;
import com.neome.api.meta.base.Types.MetaIdField;
import com.neome.api.meta.base.Types.MetaIdLayoutGrid;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioDtoLayoutGrid extends StudioBase
{
  @Nullable
  public MetaIdLayoutGrid[] allowToSwitchLayoutIdSet;

  @Nullable
  public MetaIdField bgColorFieldId;

  @Nullable
  public String description;

  public EnumDefnLayoutGridKind kind;

  @Nullable
  public String label;

  public MetaIdLayoutGrid metaId;

  public Symbol name;

  @Nullable
  public MetaIdField toolTipFieldId;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoLayoutGrid.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.Symbol
import com.neome.api.meta.base.Types.EnumDefnLayoutGridKind
import com.neome.api.meta.base.Types.MetaIdField
import com.neome.api.meta.base.Types.MetaIdLayoutGrid

interface StudioDtoLayoutGrid : StudioBase {
    val allowToSwitchLayoutIdSet: List<MetaIdLayoutGrid>?
    val bgColorFieldId: MetaIdField?
    val description: String?
    val kind: EnumDefnLayoutGridKind
    val label: String?
    val metaId: MetaIdLayoutGrid
    val name: Symbol
    val toolTipFieldId: MetaIdField?
}
